jdaviz/configs/default/plugins/model_fitting/initializers.py

```diff
@@ -203,6 +222,51 @@ def _set_width_attribute(self, instance, name, fwhm):
         _setattr(instance, name, WIDTH, fwhm / 2.355)
 
 
+class _BlackBodyInitializer(object):
```

Comment:
No need for `(object)`, which is a remnant from Python 2 days:

```suggestion
class _BlackBodyInitializer:
```